game_screens/menu_music.py

```
@@ -19,70 +20,81 @@ def __init__(self, screen):
     def play_music(self):
         path = self.songs[self.current_index][1]
-        pygame.mixer.music.load(path)
-        pygame.mixer.music.play(-1)
\ No newline at end of file
+        if not os.path.exists(path):
+            print(f"[MusicMenu] missing file: {path}")
+            return
+        if not pygame.mixer.get_init():
+            try:
+                pygame.mixer.init()
+            except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)